ext-src/NexusExplorer.ts

```diff
@@ -69,7 +69,7 @@ export class NexusExplorerProvider
     });
   }
 
-  private async reloadComponentModel() {
+  private async reloadComponentModel(): Promise<any> {
     await this.componentModel.evaluateComponents();
```

Comment:
if it's async, do we need `await` here?

Reply:
async allows you to await! (I'm not sure, I'll take a look tomorrow)

Reply:
In checking, I decided to nix it, you can see that in my follow up commits.